workorder/customizable_workorder.tpl

```diff
@@ -520,6 +535,19 @@ img.barcode {
 			{% endif %}
 		</td>
 	</tr>
+	{% for WorkorderSaleLine in Line.SaleLine %}

```

Comment:
Do we not want these changes in other languages?

Reply:
Oh! Totally my mistake, good catch. I'll update the other languages